### PR TITLE
feat: lumberjack log validation cli

### PR DESCRIPTION
**Pull request by sqin2019**
part of #427

### CODE REVIEW

File: pkg/cli/root.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	"context"
+
+	"github.com/abcxyz/lumberjack/internal/version"
+	"github.com/abcxyz/pkg/cli"
+)
+
+// rootCmd defines the starting command structure.
+var rootCmd = func() cli.Command {
+	return &cli.RootCommand{
+		Name:    "ljctl",
```
Comment (sethvargo) [inline pkg/cli/root.go:28]: Hmm - we should probably name this `lumberjackctl`. `ljctl` is kinda weird.

File: pkg/cli/validate.go [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+		Aliases: []string{"l"},
+		Target:  &c.flagLog,
+		Example: `{"foo":"bar"}`,
+		Usage: `The lumberjack/data access log, in JSON format. Set the value to` +
+			` "-" to read from stdin, it stops reading when it reaches end of file`,
+	})
+
+	f.BoolVar(&cli.BoolVar{
+		Name:    "additional-check",
+		Aliases: []string{"ac"},
```
Comment (sethvargo) [inline pkg/cli/validate.go:74]: I don't think we should have an alias for this one.

File: pkg/cli/validate.go [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+	if c.flagLog == "" {
+		return fmt.Errorf("log is required")
+	}
+
+	if c.flagLog == "-" {
+		// Read log from stdin
+		log, err := c.readFromStdin(ctx, "Enter log: ")
+		if err != nil {
+			return fmt.Errorf("failed to get log from prompt: %w", err)
+		}
```
Comment (sethvargo) [inline pkg/cli/validate.go:102]: We can actually get rid of this now and call `c.Prompt()` 😄

Comment (sqin2019) [reply pkg/cli/validate.go:102]: Chen and I [discussed about this](https://github.com/abcxyz/lumberjack/pull/428#discussion_r1281193787), the existing c.Prompt() only reads one line as it uses bufio scanner.scan, whereas lumberjack logs are multi-lines. I am thinking I can add a multi-line reader here and then move it to pkg. Any suggestions?

File: pkg/cli/root.go (hunk 1 above)
Comment (sqin2019) [reply pkg/cli/root.go:28]: Cool, I am bad at naming, was thinking maybe shorter is preferred.

Comment (yolocs) [reply pkg/cli/root.go:28]: I vote for "lumberctl" :) After all, we are looking at logs (lumber).

File: pkg/cli/validate.go (hunk 3 above)
Comment (sethvargo) [reply pkg/cli/validate.go:102]: Ah good point. Then this is probably fine.

Comment (yolocs) [reply pkg/cli/validate.go:102]: _Could_ be multiple lines. I'm thinking `cat log.json | cli validate -log -` style

File: pkg/cli/root.go (hunk 1 above)
Comment (sqin2019) [reply pkg/cli/root.go]: Changed to the shorter version lumberctl, let me know if any concerns. Thanks both!

**Review by yolocs — APPROVED** [review]

File: pkg/cli/root.go (hunk 1 above)
Comment (sqin2019) [reply pkg/cli/root.go]: @sethvargo friendly ping

**Review by sethvargo — APPROVED** [review]

**Pull request merged**